db/db_impl/db_impl.cc

```diff
@@ -6882,6 +6895,32 @@ void DBImpl::RecordSeqnoToTimeMapping() {
   sv_context.Clean();
 }
 
+void DBImpl::TriggerPeriodicCompaction() {
+  TEST_SYNC_POINT("DBImpl::TriggerPeriodicCompaction:StartRunning");
+  {
```

Comment:
With a fixed, infrequent interval, seems like we could log something about it every time this function is invoked, for potential diagnostic purposes.

Reply:
Added basic logging that this function is invoked, can add more diagnostic logging here in the future if needed.